python/lsst/pex/config/configurableField.py

```diff
@@ -240,9 +240,9 @@ def save(self, outfile, instance):
             ConfigClass = value.ConfigClass
             for module in set([target.__module__, ConfigClass.__module__]):
                 outfile.write(u"import {}\n".format(module))
-            outfile.write(u"{}.retarget(target={}, ConfigClass={})".format(fullname,
-                                                                          _typeStr(target),
-                                                                          _typeStr(ConfigClass)))
+            outfile.write(u"{}.retarget(target={}, ConfigClass={}\n\n)".format(fullname,
```

Comment:
Ha!  You're right.  And now I'm really confused as to why we don't see a failure from this more often.